### PR TITLE
Added Product Page

### PR DESCRIPTION
**Pull request by adithyark17**
Screenshot 2022-02-21 at 6 25 19 PM

### CODE REVIEW

File: pages/Product-Listing.html [hunk 1]
```diff
@@ -61,8 +62,94 @@ <h2>Sort</h2>
...
         <main class="contents">
-            <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ut et laboriosam dignissimos ea, sed beatae praesentium porro esse numquam ipsum eum maxime voluptatibus cupiditate illum architecto corporis, distinctio obcaecati recusandae.</p>
+            <h2>All Results</h2>
+            <div class="grid obj-two">
+                <!-- card 1  -->
+                <div class="card-container shadow">
 
+                    
+                    <div class="card-image">
+                        <img src="/Assets/081227964603.jpg" alt="">
```
Comment (aatishpandey) [inline pages/Product-Listing.html:72]: please fill the alt attribute

Comment (srejitk) [inline pages/Product-Listing.html:72]: Consider adding alt texts to images.

File: pages/Product-Listing.html [hunk 2]
```diff
@@ -61,8 +62,94 @@ <h2>Sort</h2>
...
+                        <button class="primary-btn"><a class="btn-link" href="">Add to Cart</a></button>
+                        <button class="secondary-btn"><a class="btn-link" href="">Add to Wishlist</a></button>
+                    </div> 
+                </div>
+                <!-- Card 3 -->
+                <div class="card-container shadow">
+
+                    
+                    <div class="card-image">
+                        <img src="/Assets/iu-8.jpeg" alt="">
```
Comment (aatishpandey) [inline pages/Product-Listing.html:106]: same here

File: pages/Product-Listing.html [hunk 3]
```diff
@@ -61,8 +62,94 @@ <h2>Sort</h2>
...
+        
+                    </div>
+                    <div class="card-contents">
+                        <h2>Product</h2>
+                        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque corporis cumque commodi quod incidunt sint sapiente.
+                    </div>
+                    <div class="card-action">
+                        <button class="primary-btn"><a class="btn-link" href="">Add to Cart</a></button>
```
Comment (aatishpandey) [inline pages/Product-Listing.html:114]: don't leave href empty

File: pages/Product-Listing.html (hunk 2 above)
Comment (srejitk) [inline pages/Product-Listing.html:106]: Same as above. Please do a check for all images.

File: pages/Product-Listing.html [hunk 4]
```diff
@@ -61,8 +62,94 @@ <h2>Sort</h2>
...
+                    <div class="card-image">
+                        <img src="/Assets/iu-4.jpeg" alt="">
+        
+                    </div>
+                    <div class="card-contents">
+                        <h2>Product</h2>
+                        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Atque corporis cumque commodi quod incidunt sint sapiente.
+                    </div>
+                    <div class="card-action">
+                        <button class="primary-btn"><a class="btn-link" href="">Add to Cart</a></button>
```
Comment (aatishpandey) [inline pages/Product-Listing.html:131]: same here

File: pages/Product-Listing.html [hunk 5]
```diff
@@ -9,15 +9,16 @@
 </head>
 <body>
     <header class="navbar">
-        <img src="" alt="Logo">
+        <a href="/pages/Home.html">VinylKart</a>
         <div class="search">
             <span>Search</span>
             <input type="search" name="" id="">
```
[the remark this answers is not in the document]
Comment (adithyark17) [reply pages/Product-Listing.html:15]: its removed

File: style-guide/Product-Listing.css [hunk 6]
```diff
@@ -58,8 +56,11 @@ main {
 }
 
 button {
-    padding: 0.5rem;
+    padding: 0.5rem 1rem;
+    border-radius: 5px;
     margin: 1rem;
+    background-color: var(--secondary-color);
+    color: white;
```
Comment (aatishpandey) [inline style-guide/Product-Listing.css:63]: you can use variable for color here

**Review by srejitk — APPROVED** [review]
Suggested a few changes. Looks good wise.

**Pull request merged**